src/components/standalone/firewall/nat/CreateOrEditNatRuleDrawer.vue

```diff
@@ -93,6 +94,12 @@ const actionOptions = ref([
   }
 ])
 
+const anyAddress = ref({
+  id: '0.0.0.0/0',
+  label: '0.0.0.0/0',
+  description: t('standalone.nat.any_address')
+})
```

Comment:
This can be a `dict` without ref wrapping? Standing by the code, it appears that afterwards you'll have a mixed `destinationAddressOptions` which contains both a `ref` of this and standard `dict`s